src/transformers/cache_utils.py

```diff
@@ -19,6 +19,13 @@
 logger = logging.get_logger(__name__)
 
 
+class CacheInfo:
+
+    def __init__(self, position, length):
+        self.position = position
+        self._length = length
```

Comment:
Introduce this class to encapsulate all the information (`position`, `length`) instead of passing those as separate arguments.